ui/components/multichain/account-overview/account-overview.tsx

```diff
@@ -38,7 +38,7 @@ export function AccountOverview(props: AccountOverviewProps) {
 
   return (
     <>
-      {!useExternalServices && (
+      {!process.env.REMOVE_GNS && !useExternalServices && (
```

Comment:
Does this PR relate to removing the GNS at all?  Can this carousel item just go in?